src/components/input/index.tsx

```diff
@@ -183,16 +183,16 @@ export class SmoothlyInput {
 	}
 	render() {
 		return (
-			<Host class={{ "has-value": this.state?.value != "" }}>
+			<Host class={{ "has-value": this.state?.value != undefined }}>
```

Comment:
`.has-value` controls if the `<label/>` in smoothly-input is moved up or not. If the `state.value` is empty then label be in the input as a kindof placehover.
```suggestion
			<Host class={{ "has-value": this.state?.value != undefined && this.state?.value != "" }}>
```